tests/test_users/test_routers.py

```diff
@@ -61,9 +61,9 @@ def test_authenticated_user_can_update_user(
     get_token_header: dict[str, str],
     db_users: list[UserOutputSchema]
 ):
-    response = sync_client.put(f"users/{db_users[0].id}", json=update_data, headers=get_token_header)
-    print(response.json())
-    assert response.json()["username"] == update_data["username"]
+    response = sync_client.patch(f"users/{db_users[0].id}", json=update_data, headers=get_token_header)
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json()["email"] == update_data["email"]

```

Comment:
1.get_token_header powinien miec zmieniona nazwe, bo to nie funkcja, tylko jakis dict zdaje sie
2. przydalaby by sie jakas fixtura authenticated_user czy cos, bo db_users[0] jest nie do konca poprawne